doc/source/commands/flight/cooked.rst

```diff
@@ -84,13 +87,25 @@ The special LOCK variables for cooked steering
         which tells the steering manager to attempt to stop any vessel rotation,
         much like the stock SAS's stability assist mode.
 
+
 Like all ``LOCK`` expressions, the steering and throttle continually update on their own when using this style of control. If you lock your steering to velocity, then as your velocity changes, your steering will change to match it. Unlike with other ``LOCK`` expressions, the steering and throttle are special in that the lock expression gets executed automatically all the time in the background, while other ``LOCK`` expressions only get executed when you try to read the value of the variable. The reason is that the **kOS** computer is constantly querying the lock expression multiple times per second as it adjusts the steering and throttle in the background.
 
 .. warning::
 
-    It's a very bad idea to ``WAIT`` during the execution of the expression in a
-    LOCK STEERING.  See the note in the next section below.
-
+    **About ``lock steering`` and ``SAS``**:  Both kOS's ``lock steering``,
+    and stock KSP's ``SAS`` are steering controllers that attempt to
+    compensate for "unknown" rotational forces (drag, offcenter thrust, etc)
+    and fight them by moving the controls to counter their effects when they
+    notice them happening.  The problem is that SAS is unaware that what
+    kOS is triyng to do is deliberate and it sees it as something that
+    has to be countered.  (i.e. kOS causes the ship to start rotating to the
+    left to seek a new heading, then SAS sees this rotation and not knowing 
+    that it's deliberate, it tries to fight it by steering harder to the
+    right.) The effect of this in-fighting is that if you ``lock steering``
+    while ``sas`` is on, you will end up seeing the craft spin around
+    wildly in a seemingly nonsensical way.  It is a good idea to 
+    get in the habit of doing ``sas off.`` whenever you ``lock steering``
+    to something.
```

Comment:
This isn't quite accurate as the root cause.  In the old version of the steering code this was true.  But the problem currently is that we tell SAS to point to our selected rotation, but it must not be the right rotation anymore.  I know for a fact that SAS integration worked for a time, and then it broke again.  If I had more time before release I'd try to find the actual fix, but I want to hold up publishing.  I'm not sure that the text here actually needs to change, because the explanation doesn't really help the user.  If we change it, I would simply leave it at "it doesn't work, don't do it".